### PR TITLE
Update website for Fall 2024

### PR DESCRIPTION
**Pull request by taylorgibson**

### CODE REVIEW

**Comment by taylorgibson** [comment]
@rowanc1 Does this seem okay? If so, I can merge.

**Comment by fwkoch** [comment]
Hey @taylorgibson - Just taking a quick look at this - I'll set up a github action to auto-deploy this site. That way, when you merge this, the site should update.

**Comment by fwkoch** [comment]
Ok - let's get this one in first: https://github.com/ncssm/morsci-website/pull/3 - then I'll keep an eye on things and make sure deploys go smoothly without extra intervention!

**Review by fwkoch — APPROVED** [review]
Small changes, all look good!

File: index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: The Morganton Scientific
+title: Morganton Scientific
 description: The Morganton Scientific, NCSSM's official journal of student STEM research
 ---
 
-Welcome to the The Morganton Scientific, NCSSM's official journal of student STEM research!
+Welcome to the *Morganton Scientific*, NCSSM's official journal of student STEM research!
```
Comment (rowanc1) [inline index.md]: This is the content on the main page under the articles. You mentioned you might not want that?

**Review by rowanc1 — APPROVED** [review]
@taylorgibson Go for the merge, the changes will be shown now after a few moments.

File: index.md (hunk 1 above)
Comment (taylorgibson) [reply index.md]: That's correct. I will remove this line and the following line completely if that will remove the text from under the articles.

**Pull request merged**